languages/cpp/CMakeBuild.md

```diff
@@ -1,33 +1,49 @@
-# CMAKE build
+# CMake Build
 
-## INTRODUCTION
+## Introduction
 
-Cmake is used to build the c++ Bitwarden client library. Output should be placed in the build directory. The output contains two dynamic libraries: one that we are building `BitwardenClient` and another that the building library uses `bitwarden_c`.
+Cmake is used to build the C++ Bitwarden client library. Output should be placed in the build directory.
 
-## PREREQUISITES
+The output contains two dynamic libraries:
+
+- The C++ client `BitwardenClient`
+- The Bitwarden library used by the C++ client `bitwarden_c`.
+
+## Prerequisites
 
 - Cmake installed, minimum version 3.15
 - `schemas.hpp` generated into `include` directory
 - installed `nlohmann-json` library
 - installed `boost` library
 
-## BUILD commands
+## Build Commands
 
-One should be in the root directory of the c++ wrapper (the same level where is CMakeLists.txt placed). Paths of the three libraries should be placed inside the cmake build command:
+One should be in the root directory of the C++ wrapper (the same level where is CMakeLists.txt placed). Paths of the three libraries should be placed inside the cmake build command:
 
-$ mkdir build
-$ cd build
-$ cmake .. -DNLOHMANN=/path/to/include/nlohmann -DBOOST=/path/to/include/boost -DTARGET=relative/path/to/libbitwarden_c
-$ cmake --build .
+```bash
+mkdir -p build
+cd build
+cmake .. -DNLOHMANN=/path/to/include/nlohmann -DBOOST=/path/to/include/boost -DTARGET=relative/path/to/libbitwarden_c
+cmake --build .
+```
 
+## Example
 
+### macOS
```

Comment:
This is technically an example of the `## Build Commands`, so it should be nested ?
Also the `## Example` and `### macOS` could become one ,like `### Build example on macOS`

Reply:
This is an example of both install the prerequisites and the build commands.

It shouldn't be nested under just build commands.

I'm a fan of the following structure when we go to add other OS examples.
```
## Example 
### macOS
### Linux
### Windows
```